src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -2029,36 +2029,34 @@ private static object NullRule(CallSite site, object obj)
 
```

Comment:
@vexx32, your last commit had 3 failures in `PowerShell-CI-macos`
TabCompletion.Miscellaneous completion tests.Test history completion
```powershell
Expected strings to be the same, but they were different.
Expected length: 23
Actual length:   598
Strings differ at index 0.
Expected: 'Test history completion'
But was:  '$env:POWER...'
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 909
909:             $res.CompletionMatches[0].CompletionText | Should -BeExactly "Test history completion"
```
Test-Connection.Ping.Force IPv4 with implicit PingOptions
```powershell
Expected exactly '10.79.9.126', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 84
84:             $result[0].Address | Should -BeExactly $realAddress
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.11.249', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result[0].TargetAddress | Should -BeExactly $realAddress
```